k8s/crd-samples/pool-resource-cluster-1.yaml

```diff
@@ -1,13 +1,17 @@
 apiVersion: openebs.io/v1alpha1
 kind: CStorPool
 metadata:
-  name: pool1
+  name: pool1-abc
   node: node-host-label
+  labels:
+    "kubernetes.io/hostname": "node-host-label"
+  finalizers: ["openebs"]
 spec:
   disks:
    diskList: ["/dev/sdb1"]
   poolSpec:
-   poolName: pool1
    cacheFile: /tmp/pool1.cache
-   poolType: mirror
-
+   poolType: striped
+   overProvisioning: false
+status:
+  phase: init
```

Comment:
Do we set the phase?
The status should be set by the operator which manages this specification.
Status should never be set on the yaml manually.

Reply:
will make the comments reg the same in yaml. maya should provide some Owner/admission controller for metadata and status fields.